### ISSUE TITLE
Question about the config files and model structure

### ISSUE THREAD

**Issue by tuanho27** [luminxu/Pose-for-Everything#9]
Hi there,

Congrat and thank you for your great work.
After playing with the dataset, I finally can train both of the configs that you proposed; however, compared with the model structure in your paper, it has two configs for POMNet and Matching separately. Is it true to understand that you trained the KIM and MH modules independently, if that, how did you combine them for evaluation in table 3. Could you share?
![image](https://user-images.githubusercontent.com/22056468/186566542-1840ddc2-957f-4b0c-af90-733701fdc33a.png)

Thanks
Tuan HL

**Comment by jin-s13** [comment]
POMNet includes KIM and MH, and these are trained together.

**Comment by tuanho27** [comment]
Hi, Thanks for your quick response and clarify. 
As I see from the config files, it is separated. Sorry but am I missing something or you have not released | keep the config to train both modules together?

![questionPomnet](https://user-images.githubusercontent.com/22056468/186576132-0e6f1f43-3cf5-4269-a9a6-d34413be79e5.png)


Thanks

**Comment by kltrock** [comment]
The KIM and MH are defined [here] (https://github.com/luminxu/Pose-for-Everything/blob/main/pomnet/models/keypoint_heads/transformer_head.py)
If you go through the code during training, you can see them. 

Did you generate the whole dataset, Tuan? If so, could you please show me how to do it, thanks. 
And are the results on the test set close to the ones reported in the paper?

**Comment by tuanho27** [comment]
Thank you for pointing it out, I see :). It is my missing, the config makes me confuse a bit. I've tried to update the data, train, and quick evaluations but do not dive in through the training code. 

For the dataset, yes I try to collect and adjust one by one, all is ok except for DeepFashion and Carfusion data, which I end up using my update coco format for them, removing the sub-category they provide and merging the new one into the author data annotation with update categories. So the evaluation is not the same as the report. The result after 40 epochs of pomnet is about ~40 PCK

**Comment by kltrock** [comment]
@tuanho27, anh Tuân ở VinAI ạ?, giờ em mới nhận ra anh :D
